types/lib/utils/flush.d.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * DO NOT EDIT
+ *
+ * This file was automatically generated by
+ *   https://github.com/Polymer/gen-typescript-declarations
+ *
+ * To modify these typings, edit the source file(s):
+ *   lib/utils/flush.html
+ */
+
+/// <reference path="boot.d.ts" />
+
+declare namespace Polymer {
+
+
+  /**
+   * Adds a `Polymer.Debouncer` to a list of globally flushable tasks.
+   */
+  function enqueueDebouncer(debouncer: Polymer.Debouncer|null): void|null;
```

Comment:
Can not be `null`. That would throw an error later in `flushDebouncers()`

Reply:
Done. Updated `@param` annotation.